doc/release_notes-20-05.txt

```diff
@@ -17,22 +17,14 @@
 Capability-based security using seccomp on Linux
 ################################################
 
-;@throwException ?
+My goal for the Genode Community Summer 2019 was to enable seccomp for base-linux to achieve an intermediate level of security for a Genode system running on Linux. To get any security benefit from seccomp, it turned out the RPC mechanisms of base-linux needed to be significantly reworked to prevent processes from forging any capabilities.
 
-commit 7bf47b19826e5d45277a4306fba28d3fd98e88f6
-Author: Norman Feske <norman.feske@genode-labs.com>
-Date:   Tue Mar 31 13:41:50 2020 +0200
+The new capability-based security on Linux maps each capability to a pair of socket descriptors, one of which can be transferred along socket connections using kernel mechanisms. Each invocation of a capability uses the received socket descriptor to address the server which in turn uses the epoll framework of the Linux kernel to get notification of incoming messages and the server side socket descriptor to securely determine the invoked RPC object. Capabilities which are passed back to the server rather than invoked can be securely identified by their inode number. This way, no client can forge any capability.
+
+With the hard part finally finished thanks to a concerted effort by Norman Feske, I could turn back to seccomp. This Linux kernel mechanism restricts the ability of a process to use syscalls. Thanks to the small interface used by Genode processes, the whitelist approach worked nicely, restricting all Genode processes to just 25 syscalls on x86, none of which can access any file on the host system. Instead all access to the host system must go through Genode RPC mechanisms to one of the hybrid components which are not yet subject to seccomp. Although some global information of the host system may still be accessed, the possibilities of escaping a sandboxed Genode process are vastly reduced.
+
+Note that these changes are transparent to any user of base-linux in all but one way: The number of available socket descriptors per process need to be increased by running ulimit -n 4096 before running Genode on Linux.
```

Comment:
Doesn't 7bf47b19826e5d45277a4306fba28d3fd98e88f6 tranparently boost the soft FD limit to the hard limit anyway?

Reply:
> Doesn't [7bf47b1](https://github.com/nfeske/genode/commit/7bf47b19826e5d45277a4306fba28d3fd98e88f6) tranparently boost the soft FD limit to the hard limit anyway?

Thanks for the hint. It escaped me, as I'm mostly working with scenarios that need an increased hard limit, too.